src/errors.rs

```diff
@@ -12,6 +14,14 @@ error_type! {
     pub enum SpaceapiServerError {
         Redis(RedisError) {
             cause;
-        }
+        },
+        IoError(io::Error) {
+            cause;
+        },
+        Message(Cow<'static, str>) {
+            desc (e) &**e;
+            from (s: &'static str) s.into();
+            from (s: String) s.into();
+        },
```

Comment:
A semantically relevant name, e.g. `ArgumentParsing`.

Btw, where does the `IoError` come from? Maybe we could wrap that too?